### PR TITLE
Including tagging for lightstep incident response on Datadog

### PR DESCRIPTION
**Pull request by Rohini-Ramakrishnappa**
### What does this PR do?

Including tags for lightstep Incident response tile on Data dog

### Motivation

Provide the customers with better ability of finding lightstep incident response tile on Datadog

### CODE REVIEW

File: lightstep_incident_response/manifest.json [hunk 1]
```diff
@@ -14,7 +14,12 @@
     "classifier_tags": [
       "Supported OS::Linux",
       "Supported OS::Mac OS",
-      "Supported OS::Windows"
+      "Supported OS::Windows",
+      "Category::Alerting",
+      "Category::Collaboration",
+      "Category::Issue Tracking",
+      "Category::Notification",
+      "Category::Incident response"
```
Comment (steveny91) [inline lightstep_incident_response/manifest.json:22]: I don't think this classifier tag exist. I think the closest one we have is `"Category::Incidents"`. Can we also move it up to keep the alphabetical order.

Comment (Rohini-Ramakrishnappa) [reply lightstep_incident_response/manifest.json]: Made this change, Please review and confirm

**Comment by fanny-jiang** [comment]
Hi @Rohini-Ramakrishnappa, apologies for the delay here. There were some recent changes to our validation backend resulting in some fields in the v2 manifest being validated differently from before, in this case the `integration.metrics.check` field. Since this is a tile-only integration and it does not emit metrics, can you please try removing the `integration` key to make the validation pass? After making that change, I'll deploy it to staging to confirm that everything looks ok. Apologies for any inconvenience here--I know these validation errors are unrelated to your proposed changes.

**Comment by Rohini-Ramakrishnappa** [comment]
> Hi @Rohini-Ramakrishnappa, apologies again--seems there are different validation errors this time. Can you please try reverting back to this commit [a3d3a44](https://github.com/DataDog/integrations-extras/commit/a3d3a44fed08ed8ce4b85f1f1ad3fd2cff69d485) and changing [line 41](https://github.com/DataDog/integrations-extras/pull/1340/commits/a3d3a44fed08ed8ce4b85f1f1ad3fd2cff69d485#diff-55107b0b4adcaf1b390017f299c9960b4fda9ef3f7ca903f764087f31e132e51R41) from an empty string to an empty list?
> 
> ```
> "check": [],
> ```

@fanny-jiang  I have made the changes and the validations are now passing.  Could you please help us merge the PR.

**Comment by Rohini-Ramakrishnappa** [comment]
> > Hi @Rohini-Ramakrishnappa, apologies again--seems there are different validation errors this time. Can you please try reverting back to this commit [a3d3a44](https://github.com/DataDog/integrations-extras/commit/a3d3a44fed08ed8ce4b85f1f1ad3fd2cff69d485) and changing [line 41](https://github.com/DataDog/integrations-extras/pull/1340/commits/a3d3a44fed08ed8ce4b85f1f1ad3fd2cff69d485#diff-55107b0b4adcaf1b390017f299c9960b4fda9ef3f7ca903f764087f31e132e51R41) from an empty string to an empty list?
> > ```
> > "check": [],
> > ```
> 
> @fanny-jiang I have made the changes and the validations are now passing. Could you please help us merge the PR.

@fanny-jiang  any update on the PR review ?

**Review by alopezz — APPROVED** [review]
@Rohini-Ramakrishnappa This looks good to go!

**Pull request merged**